travvy-server/src/models/Attractions.js

```diff
@@ -0,0 +1,20 @@
+module.exports = (sequelize, DataTypes) =>
+  sequelize.define('Attractions', {
+    attraction_id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true
+    },
+    attraction_name: DataTypes.TEXT,
+    location_id: DataTypes.INTEGER,
+    city: DataTypes.TEXT,
+    country: DataTypes.TEXT,
+    email_address: DataTypes.TEXT,
+    phone_number: DataTypes.TEXT,
+    street_address: DataTypes.TEXT,
+    opening_date: DataTypes.TEXT,
+    closing_date: DataTypes.TEXT,
+    max_number_guest: DataTypes.TEXT,
```

Comment:
could this be changed to INTEGER? in the database as well.

Reply:
@rushanniazi